Library/List/SubscriptionAddOnList.cs

```diff
@@ -64,7 +66,7 @@ internal override void ReadXml(XmlTextReader reader)
         public void Add(AddOn planAddOn, int quantity = 1)
         {
             int amount = 0;
-            if (planAddOn.UnitAmountInCents.Count > 0 && !planAddOn.UnitAmountInCents.TryGetValue(_subscription.Currency, out amount))
+            if (_subscription != null && planAddOn.UnitAmountInCents.Count > 0 && !planAddOn.UnitAmountInCents.TryGetValue(_subscription.Currency, out amount))
```

Comment:
Again, client side validation is bad.

Reply:
Since this method takes in a plan add on, I'm probably going to change it to throw a similar error as the other Add methods that take a planAddOn